src/controllers/expenses.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const expenseService = require('../services/expenses.js');
+const userService = require('../services/users.js');
+
+const getAllExpenses = (req, res) => {
+  const filteredExpenses = expenseService.getAllfilterExpenses(req.query);
+
+  res.send(filteredExpenses);
+};
+
+const getOne = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expenseService.getById(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+const add = (req, res) => {
+  const data = req.body;
+
+  const userExpense = userService
+    .getAll()
+    .find(({ id }) => id === data.userId);
+
+  if (!userExpense) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newExpense = expenseService.create(data);
+
+  res.statusCode = 201;
+  res.send(newExpense);
```

Comment:
Make it in all cases
```suggestion
  res.status(201).send(newExpense);
```